pkg/operator/controllers/dnsmasq/dnsmasq.go

```diff
@@ -173,6 +230,11 @@ func ignition2Config(clusterDomain, apiIntIP, ingressIP string, gatewayDomains [
 		return nil, err
 	}
 
+	nmDispatcherRestartDnsmasq, err := nmDispatcherRestartDnsmasq()
```

Comment:
before this, drop in a feature flag above this on whether or not we enable this.